lib/stupidedi/transaction_sets/004010/implementations/IN810.rb

```diff
@@ -0,0 +1,191 @@
+                                                      b::Element(e::Required,    "Date"),
+                                                      b::Element(e::Required,    "Invoice Number"),
+                                                      b::Element(e::Situational, "Date"),
+                                                      b::Element(e::Situational, "Purchase Order Number"),
+                                                      b::Element(e::Situational, "Release Number"),
+                                                      b::Element(e::Situational, "Change Order Sequence Number"),
+                                                      b::Element(e::Situational, "Transaction Type Code"),
+                                                      b::Element(e::Situational, "Transaction Set Purpose Code"),
+                                                      b::Element(e::Situational, "Action Code"),
+                                                      b::Element(e::Situational, "Invoice Number")),
+                                           b::Segment(30, s:: REF, "Reference Identification", r::Situational, d::RepeatCount.unbounded,
+                                                      b::Element(e::Required,    "Reference Identification Qualifier"),
+                                                      b::Element(e::Situational, "Reference Identification"),
+                                                      b::Element(e::Situational, "Description"),
+                                                      b::Element(e::Situational, "Description")),
+                                           d::LoopDef.build("N1", d::RepeatCount.bounded(200),
+                                                            b::Segment(110, s:: N1, "Name", r::Situational, d::RepeatCount.bounded(1),
+                                                                       b::Element(e::Required,    "Entity Identifier Code"),
+                                                                       b::Element(e::Situational, "Name"),
+                                                                       b::Element(e::Situational, "Identification Code Qualifier"),
+                                                                       b::Element(e::Situational, "Identification Code"),
+                                                                       b::Element(e::Situational, "Code describing entity relationship"),
+                                                                       b::Element(e::Situational, "Code identifying an organizational entity, a physical location, property or an individual")),
+                                                            b::Segment(120, s:: N2, "Address Information", r::Situational, d::RepeatCount.bounded(2),
+                                                                       b::Element(e::Situational,    "Name"),
```

Comment:
The beginning of each argument should be aligned vertically with the previous line.